content/kubermatic/master/installation/install_kubermatic/_index.en.md

```diff
@@ -217,7 +219,7 @@ kubectl apply -f charts/kubermatic/crd/
 After this, the operator chart can be installed like the previous Helm charts:
 
 ```bash
-helm upgrade --tiller-namespace kubermatic --install --values YOUR_VALUES_YAML_PATH --namespace kubermatic charts/kubermatic-operator/
+helm upgrade --tiller-namespace kubermatic --install --values YOUR_VALUES_YAML_PATH --namespace kubermatic kubermaticoperator charts/kubermatic-operator/
```

Comment:
Please keep it  as `kubermatic-operator` with a dash. The fluentbit vs. fluent-bit confusion was bad enough already ^^

Reply:
Perfect, better with the dash.